packages/elements/src/components/ino-nav-drawer/ino-nav-drawer.tsx

```diff
@@ -67,9 +66,21 @@ export class NavDrawer implements ComponentInterface {
   @Prop() a11yLabels?: NavDrawerLabels = {
     content: 'Main Navigation',
     footer: 'Footer Navigation',
-    toggleBtn: 'Toggle Navigation'
+    toggleBtn: 'Toggle Navigation',
   };
 
+  /**
+   * If true, the mobile drawer will be shown. Automatically sets the variant to `modal`.
+   */
+  @Prop() isMobile?: boolean = false;
+
+  // set the variant based on the isMobile prop
+  @Watch('isMobile')
+  handleIsMobileChange() {
+    if (this.isMobile) {
+      this.variant = 'modal';
+    }
+  }
```

Comment:
What's the need for changing the `variant` property? As far as I can see the `variant` prop only applies different classes in the `render` method. The same can be achieved by also adding the classes when the `isMobile` class is set. 

The fact that the `isMobile` makes the `variant` prop obsolete when it is set to `true` might also be an indicator that the mobile view is just another `variant`. What do you think about adding `mobile` as another variant?